pkg/version/util.go

```diff
@@ -6,7 +6,8 @@ import (
 
 var (
 	validCurrentVersions = map[string]bool{
-		"1.9.0": true, "1.10.0": true, "1.11.0": true,
+		"1.10.0": true, "1.11.0": true, "1.12.0": true,
```

Comment:
Will it be good if we automate this something like below:
```go
validCurrentVersions = func(version string) bool {
if version < currentVersion {
 return true
}
return false
}
```
We can take this next release(So that we will have more test cycles)

Reply:
Sure @mittachaitu We need to verify different scenarios before going with this change.